### PR TITLE
Bah 0994 18159

### PR DESCRIPTION
**Pull request by shawkey-daniel**
## Description
VET TEC: Update Work Experience Question Answer Options#18159
As an applicant, I should not be able to answer the question “Which area best describes your high-tech work experience? (Check all that apply.)” with both a work experience option and "None of these" at the same time, so that the question can be answered correctly.

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/18159

## Testing done

- [x] local testing

- [x] unit testing

## Screenshots
image

## Acceptance criteria
- [X] "None of these" is not an answer option for the question: “Which area best describes your high-tech work experience? (Check all that apply.)”
- [X] If the applicant is not shown the question (because it is not relevant based on a previous answer), it is 'N/A' on the spool file.
- [X] If the applicant intentionally leaves a question blank (once shown), it is blank on the spool file.
- [X] If the applicant answers the question, the answers are included in the spool file.

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

Branch: bah-0994-18159

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Comment by shawkey-daniel** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/edu-benefits/0994/pages/highTechIndustry.js [hunk 1]
```diff
@@ -53,8 +56,7 @@ export const uiSchema = {
     },
     highTechnologyEmploymentType: {
       'ui:title': ' ',
-      'ui:description':
-        'Which area best describes your high-tech work experience? (Check all that apply.)',
+      'ui:description': highTechnologyEmploymentTypeDescription,
```
Comment (jbalboni) [inline src/applications/edu-benefits/0994/pages/highTechIndustry.js]: This now results in two legends in the markup for one fieldset, though one is empty. I think that's ok for now, but I need to go and figure out what the right pattern is checkbox group labels and update the forms library to support this better.

**Comment by jbalboni** [comment]
@shawkey-daniel @dmccurdybah What are you envisioning happening for in progress applications that selected the `noneApply` option, now that that appears to be removed? It looks like it might not cause any problems, but you could consider writing a migration to remove it, if necessary.

File: src/applications/edu-benefits/0994/pages/highTechIndustry.js (hunk 1 above)
Comment (shawkey-daniel) [reply src/applications/edu-benefits/0994/pages/highTechIndustry.js]: Concur, this was a workaround.

**Comment by shawkey-daniel** [comment]
> @shawkey-daniel @dmccurdybah What are you envisioning happening for in progress applications that selected the `noneApply` option, now that that appears to be removed? It looks like it might not cause any problems, but you could consider writing a migration to remove it, if necessary.

The `noneApply` option is not captured in the output spool file, so there is nothing to migrate.

**Comment by jbalboni** [comment]
@shawkey-daniel It's not in the spool file, but it's in forms that are currently saved in the in_progress_forms table. Like I said, I don't think it'll cause problems in this case because that question isn't required and because `noneApply` is removed prior to submit, but if for some reason this question changes in the future (e.g. becomes required), you could see some unexpected behavior.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**